fw/apm.c

```diff
@@ -1327,30 +1326,45 @@ __tfw_apm_crc32_calc(TfwMsgIter *it, TfwStr *chunk , struct sk_buff *skb_head,
 }
 
 bool
-tfw_apm_hm_srv_alive(int status, TfwStr *body, struct sk_buff *skb_head,
-		     void *apmref)
+tfw_apm_hm_srv_alive(TfwHttpResp *resp, TfwServer *srv)
 {
-	TfwApmHM *hm = READ_ONCE(((TfwApmRef *)apmref)->hmctl.hm);
+	int status = resp->status;
+	TfwStr *body = &resp->body;
+
+	TfwApmHM *hm = READ_ONCE(((TfwApmRef *)(srv->apmref))->hmctl.hm);
+
 	u32 crc32 = 0;
 	TfwMsgIter it;
 	TfwStr chunk = {0};
 
 	BUG_ON(!hm);
+       /*
+       * If hm->codes exists
+       * and 'status' has not in hm->codes.
+       * return NOT ALIVE
+       */
```

Comment:
I believe this is an unnecessary comment as it simply repeats what is already clearly expressed in the code.

Reply:
"Regarding tfw_http_hm_control() - we return from the function if the server is alive (not suspended) since we do not need to mark it alive with tfw_srv_mark_alive(), so the code seems correct. This is an example why it's bad to write code without comments :) "
https://github.com/tempesta-tech/tempesta/pull/1983#pullrequestreview-1663079775

But OK, commit removed.

Reply:
Well, then this should be your comment. Describe the actual meaning of what's done in the code, not just rephrase the code with words.